src/adapter/src/coord/command_handler.rs

```diff
@@ -13,6 +13,7 @@
 use std::collections::BTreeMap;
 use std::sync::Arc;
 
+use mz_sql::vars::OwnedVarInput;
```

Comment:
Can you move this down to the other `mz_` crates.

Reply:
Done. would it make sense to add [group_imports: StdExternalCrate](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#StdExternalCrate%5C%3A) to rustfmt?

Reply:
That's a good idea. Would it work with our current setup of having 4 import groups, 1) Std 2) External 3) Mz 4) Crate? If so, then yes. If not, you may want to get some opinions in the eng-general slack channel if people want to change the format of our imports.

Reply:
Ok I'll try that out in another PR

Reply:
Actually I missed that this was a unstable option. I'll wait til it gets stabilized